src/misc/mmio.c

```diff
@@ -176,23 +176,26 @@ static complex float* load_cfl_internal(const char* name, unsigned int D, long d
 		return load_zcoo(name, D, dimensions);
 
 
+	char err_str[1024];
+	sprintf(err_str, "Loading cfl file %s", name);
```

Comment:
That would overrun the stack if the name is too long which I try to test for below. How about we make io_error take format strings just like error? Then no string handling is required.